package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
@Humnaa 

can you please verify if `axios` being used. If not you can delete it from here as well.

Reply:
I'[ve removed axios because we are using it

Reply:
I've removed axios because we are not using it